webserver/webserver/src/test/java/io/helidon/webserver/utils/SocketHttpClient.java

```diff
@@ -277,7 +278,7 @@ public String receive() throws IOException {
 
             if ("".equalsIgnoreCase(t) && contentLength >= 0) {
                 char[] content = new char[contentLength];
-                br.read(content);
+                socketReader.read(content);
```

Comment:
Not entirely paying attention but since this PR crossed my desk, and knowing not very much about the overall context: whenever you call `Reader#read(char[])`, it returns an `int` representing the bytes actually read.  This invocation&mdash;again, no context, just Java developer spidey senses triggering&mdash;throws that number away, so this read may have not completed entirely.  Maybe that's OK and maybe it's not.